scripts/release.sh

```diff
@@ -14,7 +25,10 @@ fi
 if [ "$OLD_VERSION" != "none" ]; then
 	comparison="$OLD_VERSION..HEAD"
 fi
-pre_release="-p"
+
+if [ "$USE_RELEASE" != "1" ]; then
+	pre_release="-p"
```

Comment:
If the `github-release` command does not indicate this in its output, can we have an `echo` here which indicates a pre-release is being done?